man/Readme.md

```diff
@@ -7,12 +7,12 @@ To generate documentation:
 
 1) Start by filling out the 'man-template.md'
 
-2) Then use a program ronn. [Get ronn from github.](https://github.com/rtomayko/ronn)
+2) Then install a program ronn. [Get ronn from github.](https://github.com/rtomayko/ronn)
 
-3) Run ronn:
+3) Run make:
 
 ```
-$ ronn <filename>.md
+$ make -C .. docs
```

Comment:
@pabs3 
Since the build result is not reproducible, should we use `make man/git-<command>.{1,htmll}` instead of fully updating?